docs/_snippets/features/ui-language.html

```diff
@@ -1,3 +1,3 @@
 <div id="snippet-ui-language">
-	<p>This is <a href="https://ckeditor.com">CKEditor&nbsp;5</a>.</p>
+	<p>Un&nbsp;<strong>lenguaje</strong>&nbsp;(del&nbsp;<a href="http://es.wikipedia.org/wiki/Idioma_occitano">provenzal</a>&nbsp;<em>lenguatge</em>&nbsp;y este del&nbsp;<a href="http://es.wikipedia.org/wiki/Lat%C3%ADn">lat&iacute;n</a>&nbsp;<em>lingua</em>) es un sistema de&nbsp;<a href="http://es.wikipedia.org/wiki/Comunicaci%C3%B3n">comunicaci&oacute;n</a>&nbsp;estructurado para el que existe un&nbsp;<a href="http://es.wikipedia.org/wiki/Significado">contexto</a>&nbsp;de uso y ciertos principios combinatorios formales. Existen contextos tanto naturales como artificiales.</p>
```

Comment:
There's a problem here – this content doesn't load well into the editor. There's a bug that &nbsp's are being lost (i.e. removed completely). I'm going to research it a bit.

Reply:
Ok, this is https://github.com/ckeditor/ckeditor5-engine/issues/404 again. I asked @jodator to look into it. I'll remove nbsps from this HTML in the meantime.

Reply:
I've checked how this is loaded after #404 (PR soon) and it will be fixed to: `'<p>Un\u00a0<strong>lenguaje</strong>\u00a0(del\u00a0<a...`.